tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -87,18 +87,25 @@ type StringModule() =
 
     [<Test>]
     member this.Filter() =
-        let e1 = String.filter (fun c -> true) "foo"
-        Assert.AreEqual("foo", e1)
+        let e1 = String.filter (fun c -> true) "Taradiddle"
+        Assert.AreEqual("Taradiddle", e1)
 
         let e2 = String.filter (fun c -> true) null 
         Assert.AreEqual("", e2)
 
-        let e3 = String.filter (fun c -> c <> 'o') "foo bar"
-        Assert.AreEqual("f bar", e3)
+        let e3 = String.filter (fun c -> c <> '\'') "Golliwog's cakewalk"
```

Comment:
Probably a different string here.

Reply:
You don't like me to introduce an error in a famous classical piece? :p. I'll choose something else :)

Reply:
@KevinRansom I changed it into a pangram and filtered for uppercase chars, maybe that's a more sensible choice for the test: https://github.com/dotnet/fsharp/pull/9509/commits/c7d4cda9e8a22504e35e21228d1ed1601d5d131d